partiql-ast/src/pretty.rs

```diff
@@ -697,60 +697,32 @@ impl PrettyDoc for Struct {
         D::Doc: Clone,
         A: Clone,
     {
-        let wrapped = self.fields.iter().map(|p| unsafe {
-            let x: &'b StructExprPair = std::mem::transmute(p);
```

Comment:
Curious why these unsafe + transmute calls were there to begin with? Perhaps it's faster?